### ISSUE TITLE
420629:39:16.000 is flagged as invalid

### ISSUE THREAD

**Issue by palemieux** [skynav/ttt#191]
The following fails with _[12,97]:Invalid end value '420629:39:17.000'._

```
<tt xmlns="http://www.w3.org/ns/ttml" xmlns:tts="http://www.w3.org/ns/ttml#styling" xml:lang="en" >
 <head >
    <styling >
        <style xml:id="defaultStyle" tts:color="#FFFFFF" tts:textAlign="center" />
        <style xml:id="bgBlack" tts:backgroundColor="#000000" />
    </styling>
    <layout >
        <region xml:id="region1" tts:extent="100% 20%" />
    </layout>
 </head>
 <body style="defaultStyle" >
    <div region="region1" ><p xml:id="sub2356" begin="420629:39:16.000" end="420629:39:17.000" >
            <span style="bgBlack" >00:39:16.000</span>
        </p><p xml:id="sub2357" begin="420629:39:17.000" end="420629:39:18.000" >
            <span style="bgBlack" >00:39:17.000</span>
        </p></div>
 </body>
</tt>
```

**Comment by skynavga** [comment]
Fixed in 6dae66b978d3221be534bb39291d9d19378ff6f4.

**Comment by skynavga** [comment]
This raises the issue of whether there is a requirement to support a specific supremum (least upper bound) for time expressions in TTML. At present, the #time* features define support for "all values" which is probably not reasonable. For example, it would be reasonable to require support for time expression values that, when converted to seconds (with no fraction) are no greater than 2^32 - 1, which works out to be: 1193046:28:15.
